### PR TITLE
gsk: Add builder for Stroke

### PR DESCRIPTION
**Pull request by SeaDve**

### CODE REVIEW

File: gsk4/src/stroke.rs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+use crate::{LineCap, LineJoin, Stroke};
+
+impl Stroke {
+    // rustdoc-stripper-ignore-next
+    /// Creates a new builder-pattern struct instance to construct a [`Stroke`].
+    ///
+    /// This method returns an instance of [`StrokeBuilder`](crate::builders::StrokeBuilder) which can be used to create a [`Stroke`].
+    pub fn builder(line_width: f32) -> StrokeBuilder {
+        skip_assert_initialized!();
```
Comment (SeaDve) [inline gsk4/src/stroke.rs:11]: I'm not sure if this is the correct macro to use

Comment (bilelmoussaoui) [reply gsk4/src/stroke.rs:11]: It should be `assert_initialized_main_thread`

Comment (SeaDve) [reply gsk4/src/stroke.rs:11]: I see, thanks. I'm confused on how it is determined

Comment (bilelmoussaoui) [reply gsk4/src/stroke.rs:11]: You end up calling gsk::Stroke::new which has `assert_initialized_main_thread`

Comment (SeaDve) [reply gsk4/src/stroke.rs]: Oh, that makes sense

**Pull request merged**